Support storage tiering (#203)
harvester/docs: docs/advanced/storageclass.md

---
sidebar_position: 1
sidebar_label: Storage Class
title: ""
---

# Storage Class

> A StorageClass allows administrators to describe the **classes** of storage they offer. Different Longhorn StorageClasses might map to replicas policies, or to node schedule policies, or to disk schedule policies determined by the cluster administrators. This concept is sometimes called **profiles** in other storage systems.

Reviewer: ```suggestion
A StorageClass allows administrators to describe the **classes** of storage they offer. Different Longhorn StorageClasses might map to replica policies, or to node schedule policies, or disk schedule policies determined by the cluster administrators. This concept is sometimes called **profiles** in other storage systems.
```

> A StorageClass provides a way for administrators to describe the "classes" of storage they offer. Different Longhorn StorageClasses might map to replicas policies, or to node schedule policies, or to disk schedule policies determined by the cluster administrators. This concept is sometimes called "profiles" in other storage systems.

Reviewer 2: ```suggestion
A StorageClass allows administrators to describe the "classes" of storage they offer. Different Longhorn StorageClasses might map to replicas policies, or to node schedule policies, or to disk schedule policies determined by the cluster administrators. This concept is sometimes called "profiles" in other storage systems.
```

> ## How to Create a Storage Class

Reviewer 2: ```suggestion
## Creating a Storage Class
```

Author: Thanks a lot

You can create one or more StorageClasses from the **Advanced > Storage Classes** page.

:::note

After a StorageClass is created, nothing can be changed except `Description`.

:::

### Header Section
1. **Name**: name of the StorageClass
1. **Description** (optional): description of the StorageClass

### Parameters Tab

#### Number of Replicas

The number of replicas created for each volume in Longhorn. Defaults to `3`.

#### Stale Replica Timeout

> Determines when would Longhorn clean up an error replica after the replica's status become ERROR. The unit is minute. Defaults to `30` minutes in Harvester.

Reviewer: ```suggestion
Determines when Longhorn would clean up an error replica after the replica's status is ERROR. The unit is minute. Defaults to `30` minutes in Harvester.
```
